scripts/db_migrator.py

```diff
@@ -868,7 +868,8 @@ def common_migration_ops(self):
                 new_cfg = {**init_cfg, **curr_cfg}
                 self.configDB.set_entry(init_cfg_table, key, new_cfg)
 
-        self.migrate_copp_table()
+        if self.asic_type != "mellanox":
+            self.migrate_copp_table()
```

Comment:
Looks like this change can be platform agnostic, the change looks good for now as we don't know the behavior on other platforms. Please add a comment here so that as and when we validate this change for other platforms, we can make the changes accordingly here.

Reply:
Thanks, added the comment.